releasetool/update_check.py

```diff
@@ -14,11 +14,16 @@
 
 import time
 import pathlib
+import sys
 
-import pkg_resources
 import packaging.version
 import requests
 
+if sys.version_info < (3, 8):
+    import importlib_metadata as metadata
+else:
+    import importlib.metadata as metadata
```

Comment:
Consider dropping python 3.7 which is EoL.

Reply:
Fixed in 5ea803c2de145a06d828fbc6c93827caba801b2a